sunpy/coordinates/tests/test_frames.py

```diff
@@ -231,6 +231,17 @@ def test_hpc_low_precision_float_warning():
         hpc.make_3d()
 
 
+def test_hpc_obstime_from_observer():
```

Comment:
Worth adding a case where the observer obstime != obstime given when constructing the coordinate? I haven't looked so this may already be tested elsewhere.

Reply:
Let's punt such a test to a future PR.  I'm going to create an issue.